scripts/init_app_dir.sh

```diff
@@ -29,6 +32,24 @@ if [[ -z "${IN_NIX_SHELL}" ]]; then
 	cp -P /usr/lib/x86_64-linux-gnu/libgst* "${APP_DIR}/usr/lib/"
 	cp -r /usr/lib/x86_64-linux-gnu/gstreamer-1.0 "${APP_DIR}/usr/lib/"
 	cp -r /usr/lib/x86_64-linux-gnu/gstreamer1.0 "${APP_DIR}/usr/lib/"
+
+	echo "Bundling pcsc-lite 2.2.3..."
+	cp -L /usr/local/lib/x86_64-linux-gnu/libpcsclite.so* "${APP_DIR}/usr/lib/"
+	cp -L /usr/local/lib/x86_64-linux-gnu/libpcsclite_real.so* "${APP_DIR}/usr/lib/"
+	cp -L /usr/local/lib/x86_64-linux-gnu/pkgconfig/libpcsclite.pc "${APP_DIR}/usr/lib/"
```

Comment:
This is just so sad, but if it works...

Reply:
Isn't it copyied by `linuxdeployqt` automatically?
I think they should by `-executable=$(APP_DIR)/usr/sbin/pcscd` dependencies.

Reply:
You might be right, worth checking.

Reply:
back when i started working on this PR these libs were missing. I'll remove these 3 lines and check.

Reply:
hmm not copied automatically 
```

[2025-03-30T12:21:15.712Z] Bundling pcsc-lite 2.2.3...

[2025-03-30T12:21:15.712Z] chmod: cannot access 'tmp/linux/dist/usr/lib/libpcsclite.so*':
 No such file or directory

[2025-03-30T12:21:15.712Z] make: *** [Makefile:633: pkg/StatusIm-Desktop-
250330-120700-246b3b-pr17382-x86_64.AppImage] Error 1

script returned exit code 2
```